---
title: C6322
ms.date: 11/04/2016
ms.prod: visual-studio-dev15
ms.topic: reference
f1_keywords:
- C6322
helpviewer_keywords:
- C6322
ms.assetid: fb23d2b1-b2a0-465c-8bf5-ec039c6c7757
author: mikeblome
ms.author: mblome
manager: wpickett
ms.workload:
- multiple
ms.openlocfilehash: 250e3e57c555b015b04c5e03ac2936b4ff8539a0
ms.sourcegitcommit: 37fb7075b0a65d2add3b137a5230767aa3266c74
ms.translationtype: MT
ms.contentlocale: ko-KR
ms.lasthandoff: 01/02/2019
ms.locfileid: "53882054"
---
# <a name="c6322"></a>C6322
경고 C6322: 빈 _except 블록

 이 메시지는 _except 블록의 코드가 없는 임을 나타냅니다. 결과적으로, 예외 처리 되지 않은 상태가 될 수 있습니다.

## <a name="example"></a>예제
 다음 코드에서는이 경고를 생성합니다.

```
#include <stdio.h>
#include <excpt.h>
#include <windows.h>

void fd(char *pch)
{
   __try
     {
       // exception rasied if pch is null
       *pch= 0 ;
     }
   __except(GetExceptionCode() == EXCEPTION_ACCESS_VIOLATION)
     {
     }
}
```

 이 경고를 해결 하려면 다음 코드를 사용 합니다.

```
#include <stdio.h>
#include <excpt.h>
#include <windows.h>

void f(char *pch)
{
   __try
     {
       // exception rasied if pch is null
      *pch= 0 ;
     }
   __except(GetExceptionCode() == EXCEPTION_ACCESS_VIOLATION ?
               EXCEPTION_EXECUTE_HANDLER : EXCEPTION_CONTINUE_SEARCH)
     {
       // code to handle exception
       puts("Exception Occurred");
     }
}
```

## <a name="see-also"></a>참고 항목
 [try-except 문](/cpp/cpp/try-except-statement)